package exceptions;

public class ExceptionDemo1 {
	public static void main(String[] args) {
		int number = 20;
		
		try {
			System.out.println(number/0);
		}catch(Exception ex) {
			System.out.println("exception handled");
		}
	}

}
